department-of-veterans-affairs/va.gov-cms
# VACMS-4090: Fix required related info field bug.

## Description
See #4090 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/106804599-9db23800-6633-11eb-8c07-68d9e356324b.png)
![image](https://user-images.githubusercontent.com/2404547/106804619-a145bf00-6633-11eb-99c5-dc47bf7f1a88.png)

## QA steps
- As an administrator, go to `node/add/q_a`, and complete all fields required _outside_ of `Enable standalone Resources and support page for this Q&A.` (don't toggle it on).
- Save the form, and confirm no errors are thrown.
- Edit the form, toggle on the `Enable standalone Resources and support page for this Q&A.`, visually verify that a link widget appears for related info, click save, and confirm that error is thrown indicating the related info link field is required.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
